napari/_qt/qt_viewer_dock_widget.py

```diff
@@ -38,15 +40,16 @@ class QtViewerDockWidget(QDockWidget):
 
     def __init__(
         self,
-        viewer,
+        qt_viewer,
         widget: QWidget,
         *,
         name: str = '',
         area: str = 'bottom',
         allowed_areas: Optional[List[str]] = None,
         shortcut=None,
     ):
-        self.viewer = viewer
+        self.qt_viewer = qt_viewer
+        self.viewer = qt_viewer.viewer  # convenience access to viewer model
```

Comment:
Ooh, hey, a network motif! =P

Is this attribute used anywhere? I'm surprised there's no changes referring to it in this PR. If not, then maybe this specific line can wait until the PR that needs it?

Reply:
> a network motif! =P

TIL!

I was kinda making the viewer attribute available for external usage in stuff like magicgui...  but it's really not _that_ hard to just say `widget.qt_viewer.viewer`, so I'll take that alias out.